{
	"schema_version": "1.6.0",
	"id": "MAL-2024-5645",
	"modified": "2024-10-24T01:01:59Z",
	"published": "2024-06-25T13:39:29Z",
	"aliases": [
		"SNYK-PYTHON-PYBETTERASCII-5295931"
	],
	"summary": "Malicious code in pybetterascii (PyPI)",
	"details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
	"affected": [
		{
			"package": {
				"ecosystem": "PyPI",
				"name": "pybetterascii",
				"purl": "pkg:pypi/pybetterascii"
			},
			"versions": [
				"1.0"
			],
			"database_specific": {
				"source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/pybetterascii/MAL-2024-5645.json"
			}
		}
	],
	"references": [
		{
			"type": "WEB",
			"url": "https://github.com/DataDog/malicious-software-packages-dataset"
		},
		{
			"type": "ADVISORY",
			"url": "https://security.snyk.io/vuln/SNYK-PYTHON-PYBETTERASCII-5295931"
		},
		{
			"type": "ADVISORY",
			"url": "https://research.jfrog.com/malicious-packages/"
		}
	],
	"credits": [
		{
			"name": "ReversingLabs",
			"contact": [
				"https://www.reversinglabs.com"
			],
			"type": "FINDER"
		}
	],
	"database_specific": {
		"malicious-packages-origins": [
			{
				"id": "RLMA-2024-04427",
				"import_time": "2024-06-28T02:49:58.961526252Z",
				"modified_time": "2024-06-25T13:39:29Z",
				"sha256": "f9f808bf3c2f3d27ad5e422f6550092645967ad33b494112c8c7e70c4d8e2613",
				"source": "reversing-labs",
				"versions": [
					"1.0"
				]
			},
			{
				"id": "RLUA-2024-08863",
				"import_time": "2024-10-24T00:59:46.697048087Z",
				"modified_time": "2024-10-16T14:47:14Z",
				"sha256": "7ec05de80e94a202cff16a27dee5bf2eae52f49da450e0e05f020ce57a4c59c5",
				"source": "reversing-labs"
			}
		]
	}
}
